app/main/views/marketplace.py

```diff
@@ -103,8 +121,18 @@ def get_brief_by_id(framework_framework, brief_id):
     return render_template(
         'brief.html',
         brief=brief,
-        started_brief_responses_count=started_brief_responses_count,
-        completed_brief_responses_count=completed_brief_responses_count,
+        started_brief_responses_stats={
+            'started_sme_brief_responses_count': started_sme_brief_responses_count,
+            'started_large_brief_responses_count': started_large_brief_responses_count,
+            'started_brief_responses_total': started_sme_brief_responses_count + started_large_brief_responses_count
```

Comment:
@idavidmcdonald @Wynndow @samuelhwilliams 

Totally agree with your comments about the variable naming. I've changed them to be
`'sme_count', 'large_count', 'total'`
I made this decision based on the fact that they're already in a container that denotes that they are statistics about brief responses with a certain status `'completed_responses_stats', 'started_responses_stats'` so all that needs to be revealed by their key names is what differentiates them. That is that one is the count of smes, one is the count of large and one is the total.

I also changed the logic in the template to use the total for the if statement to decide whether to show the stats at all.
It was:
```
show_statistics if started_responses_stats['sme_count'] + started_responses_stats['large_count'] else do_not_show_statistics
```
It's now
```
show_statistics if started_responses_stats['total'] else do_not_show_statistics
```